woocommerce/woocommerce-gutenberg-products-block
# Use icons directly from stripe

Fixes #2064

This uses icons directly from stipe gateway, however they don't go over the same filter as the classic checkout, so we might want to account for adding filter in the future.

I went with an array of objects instead of an object since this will have arbitrary keys (name of cc).

### How to test the changes in this Pull Request:

1. In checkout see that we're using individual icons instead of a single svg.
2. weight drop.
